src/transaction/log_manager.c

```diff
@@ -4465,6 +4469,20 @@ log_append_donetime_internal (THREAD_ENTRY * thread_p, LOG_TDES * tdes, LOG_LSA
   LSA_COPY (eot_lsa, &lsa);
 }
 
+// *INDENT-OFF*
+static cubstream::stream_position log_get_group_complete_stream_position (LOG_TDES * tdes)
```

Comment:
group complete stream position should be obtained from gcm, not tdes, for both master and slave. for single node it does not matter.

Reply:
The log_append_group_complete calls are going to be changed anyway when gcms are going to be activated. I think quickfixing these log_appends temporarily without gcms would be required to tackle truncating streams after master crash & partial stream entries.